noir_stdlib/src/lib.nr

```diff
@@ -68,3 +68,11 @@ pub fn wrapping_sub<T>(x: T, y: T) -> T {
 pub fn wrapping_mul<T>(x: T, y: T) -> T {
     crate::from_field(crate::as_field(x) * crate::as_field(y))
 }
+
+#[oracle(assert_message)]
```

Comment:
Can we add some comments that this is used internally by the compiler and advise against users calling this oracle directly?

Reply:
Agreed, I'm happy to merge once this is done.